docs/incidents/2024-08-12-site-outage.md

```diff
@@ -0,0 +1,48 @@
+# Site Outage
+
+## What happened?
+
+- 2023-06-16
+  - Dependency update to `CRI-O v1.27` in production.
+- 2024-08-??
+  - AppArmor problems observed in production system logs: access denied during shutdown of containers.
```

Comment:
In particular, this apparmor denial appears to be related to:

 - https://bugs.launchpad.net/apparmor/+bug/2067900
 - https://bugs.launchpad.net/ubuntu/+source/linux/+bug/2028253